Sources/TuistSupport/Utils/FileHandler.swift

```diff
@@ -272,6 +273,10 @@ public class FileHandler: FileHandling {
         try fileManager.attributesOfItem(atPath: path.pathString)
     }
 
+    public func subpaths(_ path: AbsolutePath) -> [AbsolutePath]? {
```

Comment:
Am I right assuming that this method returns all the files/directories in the given `path`? If so, I don't think the method name is descriptive enough *(even if we are mimicking what Apple does)*. What about calling it something like: `filesAndDirectoriesContained(in path: AbsolutePath)`. It's a bit longer, but more clear.

Reply:
yep, I was trying to mimick the FileManager ones but I agree the name is really bad, will change it 🙌